### PR TITLE
Database Schema Models Work in Progress

### LINKED ISSUE #14: Implement data models
Create a data base schema document in the google drive as well.

### PR DESCRIPTION
**Pull request by anoojlal**
### Test plan

< Replace me with the steps you took to test that your changes worked correctly >  
If applicable, write Resolves #<Replace me with the issue number>

Business model and GET POST implemented

Resolves #14 #23 #24

### CODE REVIEW

File: backend/api/models/Business.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from mongoengine.fields import (
+    StringField,
+    ListField,
+    EmbeddedDocumentField,
+    EmbeddedDocumentListField
+)
+from api.models.Location import Location
+from api.models.OpenHours import OpenHours
+import mongoengine
+from mongoengine import EmbeddedDocument
```
Comment (shreyshrey1) [inline backend/api/models/Business.py]: Don't think you need to import this

File: backend/api/models/Location.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from mongoengine.fields import (
```
Comment (shreyshrey1) [inline backend/api/models/Location.py]: If you're importing one thing you don't need the parantheses
```suggestion
from mongoengine.fields import StringField
```

File: backend/api/models/OpenHours.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+)
+from mongoengine import EmbeddedDocument
+
+class OpenHours(EmbeddedDocument):
+    """Hours Embedded Document Schema"""
+
+    start = StringField(required = True)
+    end = StringField(required = True)
+    is_overnight = BooleanField(required = True)
+    day = IntField(required = True)
```
Comment (shreyshrey1) [inline backend/api/models/OpenHours.py]: Gotta make this an enum at some point

File: backend/api/views/business.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
+from flask import Blueprint
+from api.models.Business import Business
+from api.models.Location import Location
+from api.models.OpenHours import OpenHours
+from api.core import create_response, serialize_list, logger
+
+business = Blueprint("business", __name__)
+
+'''
+# function that is called when you visit /
```
Comment (shreyshrey1) [inline backend/api/views/business.py]: Delete comments later

Comment (meghatron3000) [reply backend/api/views/business.py]: Use docstrings for functions as well

**Pull request merged**